packages/react-day-picker/src/hooks/useDayEventHandlers/useDayEventHandlers.tsx

```diff
@@ -115,6 +119,12 @@ export function useDayEventHandlers(
   const onMouseLeave: MouseEventHandler = (e) => {
     dayPicker.onDayMouseLeave?.(date, activeModifiers, e);
   };
+  const onPointerEnter: MouseEventHandler = (e) => {
```

Comment:
Could we use `PointerEventHandler` for this and `onPointerLeave`?

Reply:
Updated.